velox/common/serialization/tests/SerializableTest.cpp

```diff
@@ -158,4 +159,26 @@ TEST(SerializableTest, context) {
   }
 }
 
+template <
+    template <typename, typename, typename...>
+    typename TMap,
+    typename TKey,
+    typename TMapped,
+    typename TIt,
+    typename... TArgs>
+void testMap(TIt first, TIt last) {
+  TMap<TKey, TMapped, TArgs...> map{first, last};
+  auto serialized = ISerializable::serialize(map);
+  auto copy = ISerializable::deserialize<TMap, TKey, TMapped>(serialized);
+  ASSERT_EQ(map, copy);
+}
+
+TEST(SerializableTest, map) {
+  std::vector<std::pair<int32_t, std::string>> vals{
+      {1, "a"}, {2, "b"}, {3, "c"}};
+  testMap<std::map, int32_t, std::string>(vals.begin(), vals.end());
+  testMap<std::unordered_map, int32_t, std::string>(vals.begin(), vals.end());
```

Comment:
Would you confirm that this test was failing before the changes?

Please, update the PR description to explain that ISerializable::serialize can now handle std::unordered_map and folly::F14FastMap. Otherwise, it is hard to understand what's the point of this change.